src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -57,10 +57,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
         // This is just a sample group until group creation is coordinated
-        Entity sampleGroup = new Entity(GROUP_KIND, groupID);
-        sampleGroup.setProperty(GROUP_NAME_PROPERTY, "Black Lives Matter");
-        sampleGroup.setProperty(GROUP_ID_PROPERTY, groupID);
-        datastore.put(sampleGroup);
+        Entity blmEntity = new Entity(GROUP_KIND, "123");
```

Comment:
i think you're already creating these groups in the groups servlet, so no need to create them again